src/encoded/static/scss/encoded/modules/_top_hits_search.scss

```diff
@@ -95,13 +95,15 @@
             &.section-item {
                 padding-top: 3px;
                 padding-bottom: 3px;
+                overflow-x: scroll;
```

Comment:
These can scroll horizontally? I couldn't come up with a search result to see that.

Reply:
<img width="436" alt="Screen Shot 2021-05-20 at 10 04 32 AM" src="https://user-images.githubusercontent.com/16053922/119020439-145d1200-b953-11eb-8534-a70235b14018.png">

Think it comes up if there is a long filename displayed as block.